src/test/java/com/google/devtools/build/lib/packages/util/BazelMockPythonSupport.java

```diff
@@ -39,6 +39,7 @@ public void setup(MockToolsConfig config) throws IOException {
     addTool(config, "tools/python/toolchain.bzl");
     addTool(config, "tools/python/utils.bzl");
     addTool(config, "tools/python/private/defs.bzl");
+    addTool(config, "tools/python/python_bootstrap_template.txt");
```

Comment:
note to self: this will probably need to also be added to the internal MockPythonSupport version of this class when importing